torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -579,7 +580,8 @@ at::Tensor AtenXlaType::avg_pool2d(const at::Tensor& self,
 at::Tensor AtenXlaType::avg_pool2d_backward(
     const at::Tensor& grad_output, const at::Tensor& self,
     at::IntArrayRef kernel_size, at::IntArrayRef stride,
-    at::IntArrayRef padding, bool ceil_mode, bool count_include_pad) {
+    at::IntArrayRef padding, bool ceil_mode, bool count_include_pad, c10::optional<int64_t> divisor_override) {
+  // devisor_override is set not supported yet.
```

Comment:
Look at the line below, you can add `divisor_override` for compatibility. It's not correct to ignore it.